resources/assets/sass/base/_base.colors.scss

```diff
@@ -0,0 +1,107 @@
+// Brand colors
+
+$brand-yellow: #f7ba00;
+$brand-yellow-dark: #d67d00;
+
+$brand-green: #78cc5c;
+$brand-green-dark: #288f18;
+
+$brand-red: #ed6262;
+$brand-red-dark: #cb4949;
+$brand-red-darker: #b13636;
+
+$brand-purple: #8b84d7;
+$brand-purple-dark: #5147c0;
+
+$brand-blue: #4ac9e3;
+$brand-blue-dark: #0087ae;
+
+$brand-orange: #ff9a3e;
+$brand-orange-dark: #d94d00;
+
+
+// Brand color map
+
+$colors: ('yellow' $brand-yellow $brand-yellow-dark),
+         ('green' $brand-green $brand-green-dark),
+         ('red' $brand-red $brand-red-dark),
+         ('purple' $brand-purple $brand-purple-dark),
+         ('blue' $brand-blue $brand-blue-dark),
+         ('orange' $brand-orange $brand-orange-dark);
+
+
+@mixin background-color-modifiers {
+
+	@each $name, $color, $color-dark in $colors {
+
+		&.m-#{$name} {
+
+			background: $color;
+		}
+	}
+}
+
+@mixin background-color-dark-modifiers {
+
+	@each $name, $color, $color-dark in $colors {
+
+		&.m-#{$name} {
+
+			background: $color-dark;
+		}
+	}
+}
+
+@mixin color-modifiers {
+
+	@each $name, $color, $color-dark in $colors {
+
+		&.m-#{$name} {
+
+			color: $color;
+		}
+	}
+}
+
+
+// Neutrals
+
+$color-white: #fff;
+
+$color-fill-black: #222;
+
+$color-fill-neutral-lightest: #fbfcfd;
+$color-fill-neutral-light: #f7f9fa;
+$color-fill-neutral: #e1e7eb;
+$color-fill-neutral-dark: #979da1;
+
+// $color-fill-neutral-darker: #444;
+// $color-fill-neutral-darkest: #222;
+
+$color-text-black: #444;
+
+$color-text-neutral-lightest: $color-fill-neutral-lightest;
+$color-text-neutral-light: $color-fill-neutral-light;
+$color-text-neutral: $color-fill-neutral;
+$color-text-neutral-dark: $color-fill-neutral-dark;
+// $color-text-neutral-darker: $color-fill-neutral-darker;
+// $color-text-neutral-darkest: $color-fill-neutral-darkest;
+
+
+// Shadow colors
+
+$shadow-dark: #0e1217;
+
+// Utility colors
+
+$color-text-link: #36a1eb;
+
+$color-fill-success: $brand-green;
+$color-fill-success-dark: $brand-green-dark;
+
+// New colors
+
+$color-dark-gray: #111;
```

Comment:
Just a comment: I pulled out some hardcoded gray values (mostly from mixins) which were coupled with alpha channels. At some point this needs simplification